runtime/gear/src/lib.rs

```diff
@@ -449,14 +449,30 @@ parameter_types! {
     pub const MailboxThreshold: u64 = 3000;
 }
 
+fn bank_address() -> AccountId {
+    let arr: [u8; 8] = [1, 5, 0, 8, 2, 0, 0, 1];
+    let mut addr = [0u8; 32];
+
+    for (pos, byte) in arr.into_iter().enumerate() {
+        addr[pos * 4..pos * 4 + 4].copy_from_slice(&byte.to_le_bytes());
+    }
+
+    addr.into()
+}
```

Comment:
Why don't you use something like `b"gearge...argearbank"`/`b"gearbankgearba...nkgearbank"`?

Moreover I would place this common address in `runtime/common`.

Reply:
@ekovalev advise here on entropy please. We need to be super secure